### PR TITLE
Internal: Migrate to using rush publish and rush change 

### PR DESCRIPTION
**Pull request by timotheeguerin**
Changes: 
- [x] CI will change for changelog to be included 
  - [x] except if trying to publish a new version 
- [x] set package.json versions to their current published ones

### CODE REVIEW

File: .scripts/publish.yaml [hunk 1]
```diff
@@ -11,18 +11,20 @@ pool:
...
 
   - script: |
-      # publish the packages (tag as preview by default)
-      echo "//registry.npmjs.org/:_authToken=$(azure-sdk-npm-token)" > ./.npmrc 
-      for file in common/temp/artifacts/packages/*.tgz 
-      do
-        echo "Trying to publish $file:"
-        npm publish $file --tag latest --access public --no-git-checks || echo no-worries 
-      done
+      NPM_AUTH_TOKEN="$(azure-sdk-npm-token)"
```
Comment (daviwil) [inline .scripts/publish.yaml:19]: This and `rush publish` should all be on one line

**Review by daviwil — APPROVED** [review]
Nice!

File: docs/internal/development.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  - Prettier
+  - ESLint
+  - EditorConfig
+
+## First build
+
+1. Install [rush.js](https://rushjs.io/pages/intro/get_started/) using
+
+```bash
+npm install -g rush
```
Comment (daviwil) [inline docs/internal/development.md:20]:
```suggestion
npm install -g @microsoft/rush
```

Comment (timotheeguerin) [reply docs/internal/development.md]: oops

**Pull request merged**